website/docs/r/monitor_log_profile.html.markdown

```diff
@@ -10,6 +10,8 @@ description: |-
 
 Manages a [Log Profile](https://docs.microsoft.com/en-us/azure/monitoring-and-diagnostics/monitoring-overview-activity-logs#export-the-activity-log-with-a-log-profile). A Log Profile configures how Activity Logs are exported.
 
+-> **NOTE:** It's only possible to configure one Log Profile per Subscription. If you are trying to create more than one Log Profile, an error with `StatusCode=409` will occur.
```

Comment:
it'd be good to catch this status code in the resource and throw a more helpful error message too (we should be able to do this [here](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/azurerm/resource_arm_monitor_log_profile.go#L119)) - what do you think?

Reply:
The status code itself stands for `Conflict`. The limitation could be one of the reason, but might not likely be the only reason.